.gitignore

```diff
@@ -0,0 +1 @@
+venv/
```

Comment:
Your .gitignore is not sufficient. You should be using something like https://github.com/JetBrains/kotlin/blob/master/.gitignore.